e2e/README.md

```diff
@@ -88,6 +89,31 @@ are available while running tests:
 | timeout           | Panic test binary after duration d (default 0, timeout disabled)              |
 | v                 | Verbose: print additional output                                              |
 
+## E2E for snapshot
+
+After the support for snapshot/clone has been added to ceph-csi,
+you need to follow these steps before running e2e.
+Please note that the snapshot operation works only if the kube version
+is greater than or equal to 1.17.0.
+
+- Delete snapshot CRD created by ceph-csi in rook
```

Comment:
```suggestion
- Delete Alpha snapshot CRD created by ceph-csi in rook

Check if we have any `v1alpha1` CRD created in our Kubernetes cluster

```bash
[$]kubectl get crd volumesnapshotclasses.snapshot.storage.k8s.io -o yaml |grep v1alpha1
  - name: v1alpha1
  - v1alpha1
[$]kubectl get crd volumesnapshotcontents.snapshot.storage.k8s.io -o yaml |grep v1alpha1
  - name: v1alpha1
  - v1alpha1
[$]kubectl get crd volumesnapshots.snapshot.storage.k8s.io -o yaml |grep v1alpha1
  - name: v1alpha1
  - v1alpha1

If we have Alpha CRD delete it as from Kubernetes 1.17.0+ the snapshot should be `v1beta1`
```